src/Plugin/facets/url_processor/DgiIrCustomizationsQueryString.php

```diff
@@ -145,9 +144,6 @@ public function buildUrls(FacetInterface $facet, array $results) {
 
       asort($filter_params, \SORT_NATURAL);
       $result_get_params->set($this->filterKey, array_values($filter_params));
-      if (!empty($routeParameters)) {
-        $url->setRouteParameters($routeParameters);
```

Comment:
Will just nuking this cause any other issues? Issue for qa to discovery me thinks

Reply:
the variable isn't set by anything; i genuinely think that this is just a screw-up by the maintainers of the facet module that went undiscovered because `empty($fake_variable)` returns `TRUE`, so leaving this in didn't explode anything. But it's also an unnecessary check we're doing on every facet url processor